Tests/test_NCBI_qblast.py

```diff
@@ -1,7 +1,9 @@
 # Copyright 2008-2016 by Peter Cock.  All rights reserved.
-# This code is part of the Biopython distribution and governed by its
-# license.  Please see the LICENSE file that should have been included
-# as part of this package.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
```

Comment:
Can we do the dual licensing separately, with a more explicit audit trail?

Reply:
OK, it thought because you are the author we can do this in one go.

Reply:
The main author yes, but there are half a dozen or so contributors to this file according to GitHub, so I'd prefer to do it a little more formally. I've got this fairly automated now ;)